### ISSUE TITLE
Generator update

### ISSUE THREAD

**Issue by JurijsNazarovs** [martinarjovsky/WassersteinGAN#80]
Hello,

In the lines 225-226 of the main.py should not it be errG.backward(mone) (minus sign) and not : 
 ```python3
errG = netD(fake)
errG.backward(one)
```

Similar to this (line 135): https://github.com/igul222/improved_wgan_training/blob/master/gan_mnist.py 
uses 
``` python3
gen_cost = -tf.reduce_mean(disc_fake)
```

**Comment by JurijsNazarovs** [comment]
To whomever it might be helpful:

Generator sign is different from the paper, because of #9, so, we can swipe the sign.
Discriminator sign is different, because critic does not use the sigmoid, thus the outputs of critic can be positive or negative. Network will learn what to apply.
